konan_cli/main.py

```diff
@@ -176,31 +176,104 @@ def build(image_name, config_file, dry_run, verbose):
+                                         'Authorization': f'Bearer {global_config.access_token}'})
+        if response.ok:
+            r_json = response.json()
+            global_config.token_name = r_json['token_name']
+            global_config.token_password = r_json['token_password']
+            global_config.save()
+        else:
+            # Refresh if access token is expired
+            if response.status_code == 401:
+                refresh_response = requests.get(url=f"{global_config.API_URL}/api/auth/token/refresh/",
+                                                headers={'content-type': 'application/json'})
+                if refresh_response.ok:
+                    global_config.access_token = refresh_response.json()['access']
+                    global_config.save()
+                    # Resend KCR creds request in case of access token is expired
+                    response = requests.get(url=f"{global_config.API_URL}/registry/token/",
+                                            headers={'content-type': 'application/json',
+                                                     'Authorization': f'Bearer {global_config.access_token}'})
+                    if response.ok:
+                        r_json = response.json()
+                        global_config.token_name = r_json['token_name']
+                        global_config.token_password = r_json['token_password']
+                        global_config.save()
+                else:
+                    click.echo(
+                        "Error fetching token_name and token_password for konan container registry, please try to re-login")
+                    return
+
+    client = docker.from_env()
+    client.login(username=global_config.token_name, password=global_config.token_password,
+                 registry=global_config.KCR_REGISTRY)
+
+    if image_tag:
+        try:
+            image = client.images.get(image_tag)
+        except ImageNotFound:
+            click.echo("Incorrect image provided")
+            return
+    else:
+        if LocalConfig.exists(DEFAULT_LOCAL_CFG_PATH):
+            local_config = LocalConfig(**LocalConfig.load(DEFAULT_LOCAL_CFG_PATH), new=False)
+            if local_config.latest_built_image:
+                if click.confirm("Do you want to use the latest built image?"):
+                    image = client.images.get(local_config.latest_built_image)
+                else:
+                    try:
+                        image = client.images.get(click.prompt("Image tag"))
+                    except ImageNotFound:
+                        click.echo("Incorrect image provided")
+                        return
+            else:
+                click.echo("Please run 'konan build' first")
+                return
+        else:
+            click.echo(
+                "Error reading local configs, please navigate to project file and make sure you initialized your project using init command")
```

Comment:
Maybe we can make this similar to the django error if manage.py is not found? The user doesn't know what's a local config
```
model.config.json does not exist in the current directory. Make sure you're running this command from the same directory you ran 'konan init'.
```